### ISSUE TITLE
dpu_mnist_classifier.ipynb with keras.datasets.mnist had poor accuracy

### ISSUE THREAD

**Issue by k6s4** [Xilinx/DPU-PYNQ#12]
I tried dpu_mnist_classifier.ipynb and it worked well.
But dpu_mnist_classifier.ipynb with keras.datasets.mnist ( original ipynb used mnist ) had poor accuracy. All prediction results were '1'.

Does anyone have any idea?

<img width="1111" alt="スクリーンショット 2020-09-29 19 23 03" src="https://user-images.githubusercontent.com/44548854/94546765-508b1780-0289-11eb-842b-40e70591606c.png">

<img width="1074" alt="スクリーンショット 2020-09-29 19 25 55" src="https://user-images.githubusercontent.com/44548854/94546995-9e078480-0289-11eb-83bf-5473b11d611a.png">

**Comment by yunqu** [comment]
You will need to use the correct model. Otherwise I don't think it will even work.
